testutils/mock_agent.go

```diff
@@ -28,12 +28,13 @@ import (
 	"sync"
 	"sync/atomic"
 
+	"github.com/apache/thrift/lib/go/thrift"
+
 	"github.com/uber/jaeger-client-go/thrift-gen/agent"
+	"github.com/uber/jaeger-client-go/thrift-gen/jaeger"
 	"github.com/uber/jaeger-client-go/thrift-gen/sampling"
 	"github.com/uber/jaeger-client-go/thrift-gen/zipkincore"
```

Comment:
aye since MockAgent has to implement EmitZipkinBatch until it can be removed from thrift.idl